migration/migration/src/lib.rs

```diff
@@ -108,8 +123,41 @@ pub mod pallet {
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
 
+		/// Set Tokens (GAME & PLAY) balances with the same amount as native currency has
+		#[pallet::weight(5_000_000)]
+		pub fn tokens_airdrop(origin: OriginFor<T>) -> DispatchResult {
+			ensure_root(origin.clone()).is_ok();
```

Comment:
This check doesn't deny access from non-root accounts. `is_ok` not raises an exception, but instead just returns boolean flag. You should check that flag and interrupt further extrinsic logic in case of `false` result. Or use `ensure_root(origin)?` as we usually do.